environment.yml

```diff
@@ -32,6 +32,10 @@ dependencies:
   - dask
   - distributed
 
+  # geopandas extra
```

Comment:
run `python scripts/generate_pip_deps_from_conda.py` to have these deps reflected in `requirements.txt`

(probably should have a pre-commit hook for this 😅 https://github.com/pandera-dev/pandera/issues/699)